www/include/reporting/dashboard/xmlInformations/GetXmlHost.php

```diff
@@ -61,9 +70,9 @@
 
     if ($accessHost) {
         $DBRESULT = $pearDBO->query(
-            "SELECT  * FROM `log_archive_host` WHERE host_id = "
+            "SELECT  * FROM `log_archive_host` WHERE host_id = '"
```

Comment:
Is it a good idea to update this query like that ? Because it's not working with mysql strict mode